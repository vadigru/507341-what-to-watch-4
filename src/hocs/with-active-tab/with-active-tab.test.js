import * as React from "react";
import renderer from "react-test-renderer";
import withActiveTab from "./with-active-tab.jsx";

const MockComponent = () => <div />;

const MockComponentWithActiveTab = withActiveTab(MockComponent);

it(`render withForm`, () => {
  const tree = renderer.create(
      <MockComponentWithActiveTab
        activeTab={``}
        onTabClick={() => {}}
      />, {
        createNodeMock: () => {
          return {};
        }}
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
