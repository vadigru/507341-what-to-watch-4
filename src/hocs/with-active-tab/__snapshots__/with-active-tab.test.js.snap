// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withForm 1`] = `<div />`;
